scrape_linkedin/Company.py

```diff
@@ -116,15 +125,24 @@ def life(self):
     def insights(self):
 
         # summary table containing the Insights data for % change in headcount at 6m, 1y and 2y
-        table = one_or_default(
-            self.insights_soup, '.org-insights-module__summary-table')
+        table = one_or_default(self.insights_soup,
+                               '.org-insights-module__summary-table')
 
         insights = {}
 
-        insights.update(get_info(table, {
-            '6m change': 'td:nth-of-type(2) span:nth-of-type(3)',
-            '1y change': 'td:nth-of-type(3) span:nth-of-type(3)',
-            '2y change': 'td:nth-of-type(4) span:nth-of-type(3)'
-
-        }))
+        insights.update(
+            get_info(
+                table, {
+                    '6m change': 'td:nth-of-type(2) span:nth-of-type(3)',
+                    '1y change': 'td:nth-of-type(3) span:nth-of-type(3)',
+                    '2y change': 'td:nth-of-type(4) span:nth-of-type(3)'
+                }))
         return insights
+
+    @property
+    def people(self):
+        content = one_or_default(self.people_soup,
+                                 '.org-grid__content-height-enforcer')
+        people = text_or_default(content, 'div > div > div > h2')
+        people = people.replace("employees", "").replace("alumni", "").strip()
+        return people
```

Comment:
All of the other properties return dictionaries of key/value pairs, but this returns a single string.

Even if only a single key is currently used, this should return a dictionary for consistency with every other property. It will also more easily allow adding new fields in the future, if appropriate.